$NetBSD: patch-Makefile,v 1.2 2017/01/01 16:03:38 leot Exp $

Do not ignore CPPFLAGS and do not force optimization in CFLAGS

--- Makefile.orig	2017-01-01 12:33:41.000000000 +0000
+++ Makefile
@@ -1,7 +1,7 @@
 VERSION=0.8
 
-CPPFLAGS=-DVERSION=\"${VERSION}\" -D_GNU_SOURCE
-CFLAGS+=-Wall -Wextra -g -std=c99 -O3 -pedantic
+CPPFLAGS+=-DVERSION=\"${VERSION}\" -D_GNU_SOURCE
+CFLAGS+=-Wall -Wextra -g -std=c99 -pedantic
 PREFIX?=/usr/local
 MANDIR?=$(PREFIX)/share/man
 BINDIR?=$(PREFIX)/bin
